README.md

```diff
@@ -22,6 +27,27 @@ If you aren't sure how to handle negative return values, try this table:
 
 The JS file `widechar_width.js` contains the function `widechar_wcwidth()`. This behaves the same as the C++ version.
 
+## Python usage
+
+`widechar_width.py` contains the function `wcwidth` that returns either an int or a member of the `Special` IntEnum.
+
+The values are the same as the other implementations, so you can compare them as int, or you can use it as an enum.
+
+`wcwidth` takes either a len-1 string or an int representing the codepoint (like you would get via `ord("f")`).
```

Comment:
len-1 is a bit confusing because `len()` measures byte-length, so I'd say something like
"a string containing exactly one codepoint, or the codepoint as int"